modules/administration-guide/partials/proc_configuring-minikube-github-authentication.adoc

```diff
@@ -0,0 +1,69 @@
+[id="configuring-minikube-github-authentication_{context}"]
+= Configuring Minikube with GitHub Authentication
+
+
+On Minikube, {prod-cli} provides a default OpenID Connect (OIDC) issuer, which can serve as a bridge to third party {identity-provider}, such as GitHub.
+{link-oidc-issuer} is the default OIDC issuer, preconfigured with static users.
+Configure {link-oidc-issuer} to use GitHub authentication.
+
+.Prerequisites
+
+* {prod-short} is installed on Minikube. See xref:installation-guide:installing-che-on-minikube.adoc[].
+
+
+.Procedure
+. Get Minikube IP and remember it as `_<minikube_ip>_`:
++
+----
+$ minikube ip
+----
+
+. link:https://github.com/settings/applications/new[Create an OAuth App] for your Minikube instance in GitHub. See link:https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app[GitHub documentation].
++
+[source]
+----
+Application name: Eclipse Che <1>
```

Comment:
shouldn't it be {prod-something} instead of Eclipse Che ?

Reply:
looks like variables does not work in `[source]`.